compound-comptroller-bot/src/agent.config.ts

```diff
@@ -0,0 +1,14 @@
+import { Network } from "forta-agent";
+import { AgentConfig } from "./utils";
+
+const CONFIG: AgentConfig = {
+  [Network.MAINNET]: {
+    compoundComptrollerAddress: "0x3d9819210A31b4961b30EF54bE2aeD79B9c9Cd3B",
+  },
+
+  [Network.GOERLI]: {
+    compoundComptrollerAddress: "0xD2C5a5ea72A7419A7AFC87697E60796a56F2763E",
```

Comment:
Hmm... contract still doesn't seem to be verified. ([Goerli Etherscan page](https://goerli.etherscan.io/address/0xD2C5a5ea72A7419A7AFC87697E60796a56F2763E))

Reply:
now it's verified 👍